api/app/Models/Notification.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Notifications\DatabaseNotification;
+use Illuminate\Support\Facades\Auth;
+
+/**
+ * Class Notification
+ *
+ * Note: Exists for lighthouse-php pagination
+ */
+class Notification extends DatabaseNotification
```

Comment:
Let's make Notifications soft-deletable. Not sure if we can set that here or if we need to set it on the base class somehow.

Reply:
Updated to be soft deletes. [soft delete notifications](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9694/commits/fd2f7757e5d0e82478f0ed6990dbd4792b0a74e2)